### DISCUSSION TITLE
Can macros be exported?

### DISCUSSION THREAD

**Discussion by gpiffaretti** [GoogleFeud/ts-macros#88] [Q&A] [answered]
We're planning to create a log system in a videogame and would like to be able to strip all the logging code from the game for production.
All examples define the macro locally and use it right there. My question is, can the macros be exported as normal functions so that we could use log macros all around the system?

**Comment by volenslavchev** [comment, marked as answer]
Hi! Yeah, you can export and import macros in your code as if they're normal functions!
